lib/src/index.js

```diff
@@ -20,16 +20,14 @@ function src(glob, opt) {
     throw new Error('Invalid glob argument: ' + glob);
   }
 
-  var streams = [
+  var outputStream = pipeline(
```

Comment:
Does pipeline actually work for this? I didn't think pipeline would emit any files from the end